src/data/finset/powerset.lean

```diff
@@ -92,6 +92,19 @@ by cases s; simp [powerset_len, val_le_iff.symm]; refl
   card (powerset_len n s) = nat.choose (card s) n :=
 (card_pmap _ _ _).trans (card_powerset_len n s.1)
 
+@[simp] theorem powerset_len_zero (s : finset α) : powerset_len 0 s = {∅} :=
+begin
+  ext,
+  rw [mem_powerset_len, mem_singleton, card_eq_zero],
+  refine ⟨λ h, h.2, λ h, _⟩,
+  rw h,
+  exact ⟨empty_subset s, rfl⟩,
+end
+
+@[simp] theorem powerset_len_empty (n : ℕ) {s : finset α} (h : s.card ≤ n) :
+  powerset_len (n + 1) s = ∅ :=
```

Comment:
this applies more generally if the argument to `powerset_len` is not of the form `n + 1`, but just `n`. So I would suggest changing `h` to `h : s.card < n`.